x-pack/plugins/cases/public/components/actions/tags/use_tags_action.tsx

```diff
@@ -37,16 +38,26 @@ export const useTagsAction = ({ onAction, onActionSuccess, isDisabled }: UseActi
     (tagsSelection: TagsSelectionState) => {
       onAction();
       onFlyoutClosed();
-      const casesToUpdate = selectedCasesToEditTags.map((theCase) => {
+
+      const casesToUpdate = selectedCasesToEditTags.reduce((acc, theCase) => {
         const tags = difference(theCase.tags, tagsSelection.unSelectedTags);
         const uniqueTags = new Set([...tags, ...tagsSelection.selectedTags]);
+        const uniqueTagsAsArray = Array.from(uniqueTags.values());
+
+        if (isEqual(theCase.tags, uniqueTagsAsArray)) {
+          onActionSuccess();
```

Comment:
Why do we want to call the success callback here? Would that trigger a refresh of the cases?


https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/public/components/all_cases/use_actions.tsx#L61
```
const tagsAction = useTagsAction({
    isDisabled: false,
    onAction: closePopover,
    onActionSuccess: refreshCases,
  });
```

Reply:
My mistake! I forgot to remove it.